core-bundle/assets/controllers/checkbox-controller.js

```diff
@@ -0,0 +1,70 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['checkAll', 'input'];
+
+    static values = {
+        rowGuards: {
+            type: Array,
+            default: ['input', 'label', 'button', 'a', '.operations'],
+        },
+    };
+
+    initialize() {
+        this.start = null;
+    }
+
+    rowClick(event) {
```

Comment:
[according to Stimulus](https://stimulus.hotwired.dev/reference/actions#naming-conventions)

> Avoid action names that simply repeat the event’s name, such as click, onClick, or handleClick

So maybe this should be `toggleRow` (and below should be `toggleInput`?

Reply:
See 84a62747ecfcd8845cf4034d0350c7d282cb967e